src/fsharp/absil/ilwritepdb.fs

```diff
@@ -994,6 +994,9 @@ let logDebugInfo (outfile: string) (info: PdbData) =
         fprintfn sw "      %s- [%d-%d]" offs scope.StartOffset scope.EndOffset
         if scope.Locals.Length > 0 then
           fprintfn sw "      %s  Locals: %A" offs [ for p in scope.Locals -> sprintf "%d: %s" p.Index p.Name ]
+
+        // TODO: Print scope.Imports
```

Comment:
Remove this TODO since you're validating by other means (though adding the code to dump the table here wouldn't hurt)

Reply:
Yeah, I will remove it for time being and will add as a separate PR.